packages/common/src/hardforks.ts

```diff
@@ -163,11 +163,6 @@ export const hardforksDict: HardforksDict = {
   osaka: {
     eips: [663, 3540, 3670, 4200, 4750, 5450, 6206, 7069, 7480, 7620, 7692, 7698],
   },
-  /**
-   * Description: Next feature hardfork after prague, internally used for verkle testing/implementation (incomplete/experimental)
-   * URL        : https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/osaka.md
-   * Status     : Final
-   */
   verkle: {
     eips: [2935, 4762, 6800],
```

Comment:
I think we should keep above comment since reading this file reads as Verkle being scheduled after Osaka. Also, it seems like from EIPs 2935 has to be removed.

Reply:
Should this be 7709 instead? :thinking:

Reply:
added descripitions though prauge onwards links are not available yet in the specs repo